app/views/patients/index.csv.erb

```diff
@@ -0,0 +1,8 @@
+<%=
+CSV.generate do |csv|
+  csv << Patient::CSV_EXPORT_FIELDS.keys # Header line
+  @patients.each do |patient|
+    csv << Patient::CSV_EXPORT_FIELDS.values.map { |field| patient.send(field) }
```

Comment:
I deeply love `send` but it's a little risky, since it'll run whatever you feed it.

I think instead we should use hash attributes here -- a nice gift from our friends at rails is that you can get at object attributes by passing it a symbol, so we could do something like this -- 

```rb
Patient::CSV_EXPORT_FIELDS.values.map { |field| patient[field.to_sym] }
```

That might not help for `has_alt_contact?` though, I'm not sure if this works for methods. If that winds up being the case and we totally have to go with send (probably don't spend more than like 10m fighting with it), let's use the slightly safer version, `public_send` which will choke if we try a private method.